tests/puppeteer/pages/FO/FO_product.js

```diff
@@ -0,0 +1,33 @@
+const CommonPage = require('../commonPage');
+
+module.exports = class FO_PRODUCT extends CommonPage {
+  constructor(page) {
+    super(page);
+
+    //Selectors for product page
+    this.productName = '#main h1[itemprop="name"]:nth-child(1)';
```

Comment:
Why is there a `nth-child` in the selector ? Is there more `h1[itemprop="name"]` in the page ?